src/services/api.js

```diff
@@ -0,0 +1,10 @@
+const URL = 'http://futuramaapi.herokuapp.com/api/quotes';
+
+const api = {
+    getQuotes() {
+        return fetch(`${URL}`)
```

Comment:
here you do not need a template literal you can just pass in ```URL``` as a parameter to ```fetch```.